### PR TITLE
fix(core): 14176 - broken url routing for all pages except the default

### PR DESCRIPTION
**Pull request by IlyaIzr**
We had an issue where redirect started to work before we got list of features
So that blocked user to share routes that require features

### CODE REVIEW

File: src/views/CommonView.tsx [hunk 1]
```diff
@@ -27,7 +27,8 @@ export function CommonView({
   getAbsoluteRoute: (path: string) => string;
 }>) {
   const [featureFlags] = useAtom(featureFlagsAtom);
-  const [currentLanguage] = useAtom(currentLanguageAtom);
+  // do we need to use this atom here?
```
Comment (Akiyamka) [inline src/views/CommonView.tsx]: We need to activate the languages somewhere, any suggestions for a better location?

File: src/core/app/features.ts [hunk 2]
```diff
@@ -22,4 +25,5 @@ export function setFeatures(value: BackendFeature[] | null) {
     newFeatures[ft.name] = true;
   });
   featureFlagsAtom.set.dispatch(newFeatures);
+  featuresWereAppliedAtom.setTrue.dispatch();
```
Comment (Akiyamka) [inline src/core/app/features.ts]: 1. When you need dispatch few actions - batch it.
```
     store.dispatch([
          featureFlagsAtom.set(newFeatures),
          featuresWereAppliedAtom.setTrue(),
        ]);
   ```
   
2. the function now does not simply set features, function name must be changed according to this changes

**Comment by vkozio** [comment]
This problem can be fixed by lazy loading Router in https://github.com/konturio/disaster-ninja-fe/blob/main/src/App.tsx#L7-L8
// import { Router } from '~core/router';
const { Router } = lazily(() => import('~core/router'));

We will rework routing, migrate to different lib, and this simple fix is way to go at the moment.
@Akiyamka thoughts?

**Comment by Akiyamka** [comment]
On the one hand, this could (potentially) slow down the start of the application, but on the other hand, it ensures that the application is initialized sooner. Regardless of the trade-off, I think this is a good solution because it is reliable and transparent

**Comment by IlyaIzr** [comment]
reopen to check if it works on stage as #368 not working on stage build

File: src/views/CommonView.tsx (hunk 1 above)
Comment (IlyaIzr) [reply src/views/CommonView.tsx:30]: temp resolving

Comment (IlyaIzr) [reply src/views/CommonView.tsx]: not yet, just removing unused `currentLanguage` variable

File: src/core/app/features.ts (hunk 2 above)
Comment (IlyaIzr) [reply src/core/app/features.ts]: It sets features and stores that it did. I'm not sure i need to change function name.
I've changed atom name that stores it

**Pull request merged**